frontend/src/stores/user/user.actions.js

```diff
@@ -1,13 +1,13 @@
 import axios from "axios";
 import { createAsyncThunk } from "@reduxjs/toolkit";
-import { USER_SVC_BASE_URL } from "../../utils/configs";
+import { API_URL } from "../../utils/configs";
 import { deleteToken, getToken, storeToken, storeUsername } from "../../utils/localStorageUtils";
 
 export const signupUser = createAsyncThunk(
   "users/signupUser",
   async ({ username, password }, thunkAPI) => {
     try {
-      const response = await axios.post(`${USER_SVC_BASE_URL}/api/user`, { username, password });
+      const response = await axios.post(`${API_URL}/user-api/user`, { username, password });
```

Comment:
After creating separate config files, can import here to prevent breaking local development.

Same for the other URLS in other files :D